src/test/java/api/items/ItemApiMoveExamples.java

```diff
@@ -208,10 +208,7 @@ public void cannotMoveDueToHoldingsRecordSchemasMismatching()
 
     Response postMoveItemsResponse = moveItems(itemsMoveRequestBody);
 
-    assertThat(postMoveItemsResponse.getStatusCode(), is(500));
-    assertThat(postMoveItemsResponse.getContentType(), containsString(TEXT_PLAIN));
-
-    assertThat(postMoveItemsResponse.getBody(), containsString("Can`t map json to 'Holdingsrecord' entity"));
+    assertThat(postMoveItemsResponse.getStatusCode(), is(200));
```

Comment:
This change states that this example is now successful. The name suggests it is intended to demonstrate a failure, please either rename the test or remove it if it is no longer applicable.

(Assuming that you decide to stick with changing the move API to use a different storage collection class)